### ISSUE TITLE
Add disassembly of fragment text sections to pstore-dump

### ISSUE THREAD

**Issue by paulhuggett** [SNSystems/pstore#13]
It would be incredibly useful to see a proper disassembly of a fragment's text section when debugging problems with the LLVM repo-object-writer code. When pstore is built inside LLVM there's a disassembler right there already...!

**Comment by paulhuggett** [comment]
Implemented as of commit 953151252165595f2eb84abf3a4e35ee218074ff (as well as 953151252165595f2eb84abf3a4e35ee218074ff in llvm-prepo)

**Comment by paulhuggett** [comment]
An example fragment produced by an inside-LLVM build of pstore-dump:
```
bash-3.2$ pstore-dump --hex --fragment c2db93450e69f48bc49f046cccaf6548 ./clang.db
---

- file      : 
      path : ./clang.db
      size : 0x400000
  fragments : 
      - digest   : c2db93450e69f48bc49f046cccaf6548
        fragment : 
            - type     : text
              contents : 
                  align   : 0x10
                  data    : 
                      - 0x00:   pushq   %rbp
                      - 0x01:   movq    %rsp, %rbp
                      - 0x04:   movl    %edi, -0x8(%rbp)
                      - 0x07:   movl    -0x8(%rbp), %edi
                      - 0x0a:   addl    $-0x1, %edi
                      - 0x0d:   movl    %edi, %eax
                      - 0x0f:   subl    $0x3, %edi
                      - 0x12:   movq    %rax, -0x10(%rbp)
                      - 0x16:   movl    %edi, -0x14(%rbp)
                      - 0x19:   ja      0x3e
                      - 0x1f:   movq    -0x10(%rbp), %rax
                      - 0x23:   movq    (,%rax,8), %rcx
                      - 0x2b:   jmpq    *%rcx
                      - 0x2d:   movl    $0x5, -0x4(%rbp)
                      - 0x34:   jmp     0x2b
                      - 0x39:   movl    $0x6, -0x4(%rbp)
                      - 0x40:   jmp     0x1f
                      - 0x45:   movl    $0x7, -0x4(%rbp)
                      - 0x4c:   jmp     0x13
                      - 0x51:   movl    $0x8, -0x4(%rbp)
                      - 0x58:   jmp     0x7
                      - 0x5d:   movl    $0x9, -0x4(%rbp)
                      - 0x64:   movl    -0x4(%rbp), %eax
                      - 0x67:   popq    %rbp
                      - 0x68:   retq
                  ifixups : 
                      - { section: read_only, type: 0xb, offset: 0x27, addend: 0x0 }
                  xfixups : [ ]
            - type     : read_only
              contents : 
                  align   : 0x8
                  data    : !!binary16 |
                      0000 0000 0000 0000 0000 0000 0000 0000
                      0000 0000 0000 0000 0000 0000 0000 0000>
                  ifixups : 
                      - { section: text, type: 0x1, offset: 0x0, addend: 0x2d }
                      - { section: text, type: 0x1, offset: 0x8, addend: 0x39 }
                      - { section: text, type: 0x1, offset: 0x10, addend: 0x45 }
                      - { section: text, type: 0x1, offset: 0x18, addend: 0x51 }
                  xfixups : [ ]
...
bash-3.2$ 
```
